service/controller/resource/templateignition/create.go

```diff
@@ -35,35 +35,34 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 		return microerror.Mask(err)
 	}
 
-	cm := &corev1.ConfigMap{
+	s := &corev1.Secret{
```

Comment:
Let's also see how much you can align this with the secret in the spec in this section:
https://github.com/giantswarm/giantswarm/blob/5a871c1971b75043d8545b6f3fcc7456dc14ba67/specs/ignition_operator.md#ignition-cr-design

Reply:
Check it out, should match the spec now